### ISSUE TITLE
[BUG][Accessibility]: Accessibility issues with tab component

### ISSUE THREAD

**Issue by jamiegaskell** [scottish-government-design-system/design-system#112]
## Description of the issue

The tab component on the design system currently has a few accessibility issues, including:

* The visually hidden heading "Contents" is inappropriate and does not describe the tab container. In this case, a more suitable heading might be "Further education support". This would need to change depending on the purpose.
* The container with `role="tablist"` does not have an accessible name. This should reference the heading using `aria-labelledby`.
* The tab panel container does not use `role="tabpanel"`.
* The tab panel container does not have an accessible name.
* The page is automatically scrolling upon arrowing between tabs. This could make navigating between tabs disorientating for users. This also occurs upon selecting a tab with the mouse.

As well, the `<nav>` region is not required and is not used in the ARIA Authoring Practices.

## Steps to reproduce the issue

1. Navigate onto the tabs using a screen reader
2. Observe that there are missing semantics
3. Inspect the DOM
4. Compare the code with [ARIA Authering Practices - Tabs with automatic activation](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/)

## Screenshots

<img width="446" alt="The tab elements on the page" src="https://github.com/user-attachments/assets/52400bb3-e7c2-4f50-b3eb-f8bf2c4d196a">

![DOM, the tab elements](https://github.com/user-attachments/assets/2ee91e0e-f5cf-4ff1-b50d-b17011c59148)

## Environment

- Operating system: Windows 10
- Browser: Edge
- Assistive Technology: JAWS 2023

**Comment by jsutcliffe** [comment]
Hi Jamie, thanks for pointing these out. I'm now looking into these fixes and a few other things I've noted.

**Comment by dkindnes** [comment]
Hi @jamiegaskell 
Version 3.2.1 (and beyond) should have a number of the tab issues addressed if you want to try it out.

https://github.com/scottish-government-design-system/design-system/releases
